product_docs/docs/epas/15/reference/sql_reference/02_data_types/05_boolean_type.mdx

```diff
@@ -16,3 +19,24 @@ redirects:
 EDB Postgres Advanced Server provides the standard SQL type `BOOLEAN`. `BOOLEAN` can have one of only two states: `TRUE` or `FALSE`. A third state, `UNKNOWN`, is represented by the SQL `NULL` value.
 
 The valid literal value for representing the true state is `TRUE`. The valid literal for representing the false state is `FALSE`.
```

Comment:
This is the TRUE keyword though (and FALSE and NULL), not a literal value (which would be a number or a quoted string)

We seem to be missing the actual string literals of 'true','yes','on' and '1' (and their alternatives) too and these get used a bit to cheaply coerce fields into correctness.

Reply:
@djw-m Should we just delete that second paragraph? That is, delete: "The valid literal value for representing the true state..." Or, if the second paragraph is adding anything meaningful, how would you reword?

Reply:
EDB Postgres Advanced Server provides the standard SQL type `BOOLEAN`. `BOOLEAN` can have one of only two states: `TRUE` or `FALSE`. A third state, UNKNOWN, is represented by the SQL `NULL` value.

Literal values representing the `TRUE` state include `'TRUE'`, `'true'`, `'y`'`,`'1'` and `'t'. Literal values representing `FALSE` include `'FALSE'`, `'false'`, `'n'`, `'0'` and`'f'`. There is no literal value for UNKNOWN; use `NULL`.